CHANGELOG.md

```diff
@@ -7,17 +7,19 @@
 * Added the ability to provide more detailed `Speed limit incorrect` feedback during turn-by-turn navigation. ([#2725](https://github.com/mapbox/mapbox-navigation-ios/pull/2725))
 
 ### Other changes
-
 * Fixed an issue which was causing clear map button disappearance in the example app when selecting the route. ([#2718](https://github.com/mapbox/mapbox-navigation-ios/pull/2718))
 
+### User Location
+* Created the `UserHaloCourseView` similar to `UserCourseView` for approximate location on iOS 14 during the navigation to represent user location. Allow the switch between `UserHaloCourseView` and `UserCourseView` when precise mode is changed. ([#2664](https://github.com/mapbox/mapbox-navigation-ios/pull/2664))
+
+
 ## v1.1.0
 
 ### Packaging
 
 * MapboxNavigationNative dependency was updated to v22.0.5 and MapboxCommon to v7.1.2. ([#2648](https://github.com/mapbox/mapbox-navigation-ios/pull/2648))
 
 ### User location
-
```

Comment:
Please try to be as clean as possible when doing rebase onto main (e.g. this line wasn't meant to be removed).

Reply:
Thanks for the suggestion!  I just cleaned the code to remove unnecessary change, and rebase the branch to main.